data/domain.yml

```diff
@@ -26,6 +26,7 @@ intents:
   - intent_core_here
   - intent_core_something_else
   - intent_core_other_languages
+  - intent_core_export_inventory
```

Comment:
I'm not sure if this is the right place for these, but I couldn't find any other inventory references so I put them next to the language flows since they're in the same section in the Miro board.

Reply:
I'm about to push a PR with an insights/inventory folder haha, we can move it there later